### ISSUE TITLE
Function to get and set annotations to annotations (meta-annotations)

### ISSUE THREAD

**Issue by mmc46** [natverse/rcatmaid#137]
Something similar to `catmaid_set_annotations_for_skeletons()`, `catmaid_get_annotations_for_skeletons()`and but for (meta-)annotations.

For `get`, API call /{project_id}/annotations/query-targets

I don't think this is possible at the moment.

**Comment by jefferis** [comment]
Does https://github.com/jefferis/rcatmaid/pull/130 (from Alex) satisfy this?

**Comment by mmc46** [comment]
For `set` yes. I couldn't see a `get` option, though.

**Comment by alexanderbates** [comment]
Added now to PR #130

**Comment by jefferis** [comment]
Just for reference, `catmaid_query_by_annotation` can already search for annotations annotated with a meta-annotation. Alex's new interface (should be merged) shortly is a bit more friendly though.

 ```
> catmaid_query_by_annotation("^ItoLee_Lineage$", type='ann')
         id                                   name       type skid
1  12851522                   ItoLee_Lineage: ALv1 annotation   NA
2  12852187                   ItoLee_Lineage: ALl1 annotation   NA
3  12852391         ItoLee_Lineage: new2_posterior annotation   NA
4  12856150                  ItoLee_Lineage: WEDa2 annotation   NA
5  12856498                  ItoLee_Lineage: ALlv1 annotation   NA
```

**Comment by jefferis** [comment]
Note that this was implemented in #130
